pkg/kritis/util/util.go

```diff
@@ -36,16 +60,94 @@ func GetResource(image string) *grafeas.Resource {
 	return &grafeas.Resource{Uri: GetResourceURL(image)}
 }
 
-func CreateAttestationSignature(image string, pgpSigningKey *secrets.PGPSigningSecret) (string, error) {
-	hostSig, err := container.NewAtomicContainerSig(image, map[string]string{})
+func CreateAttestation(image string, pgpSigningKey *secrets.PGPSigningSecret) (*cryptolib.Attestation, error) {
```

Comment:
`PGPSigningSecret` contains public key information. Can we refactor it to remove public key? (Could be done in a different PR).

Reply:
Will refactor in another PR